llvm/test/CodeGen/AMDGPU/insert_waitcnt_for_all.ll

```diff
@@ -0,0 +1,222 @@
+; Testing the -amdgpu-precise-memory-op option
+; COM: llc -mtriple=amdgcn -mcpu=hawaii -mattr=+amdgpu-precise-memory-op -verify-machineinstrs < %s | FileCheck %s -check-prefixes=GFX7
```

Comment:
What is COM: ?

Reply:
Comment. Some testcases in this file won't run if mcpu=hawaii. In the latest commit, the test file has been split into 2.